vscode-extension/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```

Comment:
Should this be the same license as Jetpack License in devbox?

Reply:
It is, but in the extension marketplace, there needs to be a license file at the root level of the extension project. That's why I made a separate file here.